### PR TITLE
working version

### PR DESCRIPTION
**Pull request by boutell**
This is a simple command line utility that can eat the result of copying and pasting from Namecheap (and probably, with minor edits, other DNS providers) into a google sheet, or a hand-created one, when exported as CSV. The result is a zone file, which is what DNS servers expect, and more specifically what Amazon Route 53 can import.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+if (failed) {
+  process.exit(1);
+}
+
+function fail(msg, record) {
+  console.error(`${msg}:`, record);
+  failed = true;
+}
+
+function parse (input) {
```
Comment (boutell) [inline app.js:43]: I've used this snippet in a couple places lately.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "csv-to-zonefile",
+  "version": "1.0.0",
+  "description": "Converts CSV to a zonefile just good enough to import into Amazon Route 53",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (abea) [inline package.json]: If it's under the apostrophe banner I think we should add our linter in here at least.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Run:
+
+```
+npm install -g csv-to-zone-file
+csv-to-zone-file < myfile.csv > myfile.zone
+```
+
+The CSV file must have the following columns:
+
+`Type`
```
Comment (abea) [inline README.md]: It'd be nice to have this in a table structure for easy parsing.

**Review by abea — CHANGES_REQUESTED** [review]
A few notes in preparation for publishing

**Comment by boutell** [comment]
eslint testing set up, and reformatted README.

**Review by abea — APPROVED** [review]

**Pull request merged**